### PR TITLE
Display the logged in user's full name and Avatar (image)

### PR DESCRIPTION
**Pull request by nimishamukherjee**
Reference issue #120.
Display the logged in user's full name and Avatar as obtained from GitHub.

Note:
We call api/user with the token and receive
{"fullName":"User name","imageURL":"https://avatars.githubusercontent.com/u/xxxxxxx?v=x"}

### CODE REVIEW

**Comment by nimishamukherjee** [comment]
[test]

File: src/app/header/header.component.html [hunk 1]
```diff
@@ -12,13 +12,13 @@
...
-        	<a href="#" id="header_dropdownToggle" class="dropdown-toggle" data-toggle="dropdown">
-          		<span class="pficon pficon-user"></span>
-							<div *ngIf="loggedInUser">
-								{{loggedInUser.fullName}}
-								<b class="caret"></b>
-							</div>
+      	<li *ngIf="loggedIn">
+        	<a *ngIf="loggedInUser" id="header_dropdownToggle">          								
+						<span *ngIf="!imgLoaded" class="pficon pficon-user"></span>							
+						<img (load)="onImgLoad()" src="{{ loggedInUser.imageURL }}"
```
Comment (sanbornsen) [inline src/app/header/header.component.html]: I guess, we need to update the User class with imageURL for this.

Comment (nimishamukherjee) [inline src/app/header/header.component.html]: User class already has this - 
export class User {
  fullName: string;
  imageURL: string;
}

Comment (sanbornsen) [inline src/app/header/header.component.html]: Ohh sorry!! Missed it. All good to go.

**Comment by joshuawilson** [comment]
[test]

**Pull request merged**